pkg/kubelet/types/constants.go

```diff
@@ -43,3 +43,13 @@ const (
 	LimitedSwap   = "LimitedSwap"
 	UnlimitedSwap = "UnlimitedSwap"
 )
+
+// Alpha conditions managed by Kubelet that are not yet part of the API. The
```

Comment:
This follows the advice from @liggitt around constants associated with new conditions at https://github.com/kubernetes/kubernetes/pull/110959#discussion_r926922818

Reply:
Also confirmed at https://kubernetes.slack.com/archives/C0BP8PW9G/p1658765248399249?thread_ts=1658759334.709079&cid=C0BP8PW9G

Reply:
this confused me.  so thanks for the pointer.